### PR TITLE
735

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
@thisandagain R?

File: views/template-preview/index.less [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        .btn-start-building {
+            height: @_button-height;
+            border-radius: 3px;
+            padding: 0 15px;
+        }
+    }
+
+    .app-preview {
+        position: fixed;
+        top: @header-height + @_button-height + (2 * @_padding);
```
Comment (thisandagain) [inline views/template-preview/index.less]: Ooof. Thanks for sorting that out. :smile:

File: views/detail/index.html [hunk 2]
```diff
@@ -10,17 +10,6 @@
             <p class="text-light" v-bind-html="'by _'" v-with="name: app.author.username"></p>
         </hgroup>
     </header>
-    <ul id="detail-share" v-if="isTemplate" style="background-color: {{app.iconColor}}">
-        <li><a href="" class="disabled">
-            <span class="fa fa-2x fa-fw fa-eye"></span><br>{{ 'Preview' | i18n }}</a>
```
Comment (thisandagain) [inline views/detail/index.html]: Happy to see these class names go away.

**Comment by thisandagain** [comment]
R+

**Pull request merged**